scripts/install_mt4.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
```

Comment:
`#!/usr/bin/env bash` is more preferred for better compatibility, as on OS X I've the latest bash in `/usr/local/bin`.
Then move: `set -e` to the next line.